app/createEvent/page.tsx

```diff
@@ -0,0 +1,48 @@
+import EventForm from "@/components/EventForm";
+import BackButton from "@/components/BackButton";
+import isValidEvent from "@/lib/isValidEvent";
+import { auth } from "@clerk/nextjs";
+import { Event } from "@prisma/client";
+
+export default async function CreateEvent() {
+	'use server';
+	const { userId } = auth();
+	
+	async function postEvent(event: Event) {
+		'use server';
+		const url = 'http://localhost:3000';
```

Comment:
This could be changed to a environment variable instead of a hard coded value.

Reply:
fax